tutorials/using-slurm-to-host-jupyter-notebooks/index.md

```diff
@@ -0,0 +1,298 @@
+[I 18:41:52.483 NotebookApp] The Jupyter Notebook is running at:
+[I 18:41:52.486 NotebookApp] http://google1-compute2:8080/?token=5f8bc1a29d0bfd6f3cbdbd29186359bc47f04054b9f7cf86
+[I 18:41:52.488 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
+[C 18:41:52.494 NotebookApp]
+
+    Copy/paste this URL into your browser when you connect for the first time,
+    to login with a token:
+        http://google1-compute2:8080/?token=5f8bc1a29d0bfd6f3cbdbd29186359bc47f04054b9f7cf86
+[I 18:43:57.756 NotebookApp] 302 GET /?authuser=0 (10.10.0.4) 0.82ms
+[I 18:43:57.830 NotebookApp] 302 GET /tree?authuser=0 (10.10.0.4) 1.00ms
+[W 18:44:02.879 NotebookApp] 401 POST /login?next=%2Ftree%3Fauthuser%3D0 (10.10.0.4) 1.73ms referer=https://8080-dot-3011042-dot-devshell.appspot.com/login?next=%2Ftree%3Fauthuser%3D0
+[W 18:44:54.375 NotebookApp] 401 POST /login?next=%2Ftree%3Fauthuser%3D0 (10.10.0.4) 4.27ms referer=https://8080-dot-3011042-dot-devshell.appspot.com/login?next=%2Ftree%3Fauthuser%3D0
+[W 18:45:09.963 NotebookApp] 401 POST /login?next=%2Ftree%3Fauthuser%3D0 (10.10.0.4) 4.88ms referer=https://8080-dot-3011042-dot-devshell.appspot.com/login?next=%2Ftree%3Fauthuser%3D0
+[I 18:46:29.529 NotebookApp] 302 GET /?authuser=0 (10.10.0.4) 0.70ms
+[I 18:46:29.671 NotebookApp] 302 GET /tree?authuser=0 (10.10.0.4) 0.88ms
+[I 18:46:33.473 NotebookApp] 302 POST /login?next=%2Ftree%3Fauthuser%3D0 (10.10.0.4) 1.14ms
+[I 18:48:36.214 NotebookApp] 302 GET /?authuser=0 (10.10.0.4) 0.74ms
+[I 18:49:14.889 NotebookApp] 302 GET /?authuser=0 (10.10.0.4) 0.72ms
+[I 18:50:41.854 NotebookApp] 302 GET /?authuser=0 (10.10.0.4) 0.63ms
+```
+
+## Setup an ssh tunnel to your Notebook
+
+You can access your notebook from the Cloud Shell using the Web Preview <walkthrough-web-preview-icon/> tool.
+
+1. Get the login token for your notebook
+
+Substitute your notebook's job number for *nn* in this command to get the login token for your notebook.
+
+```
+gcloud compute ssh google1-login1 --command 'cat jupyter-notebook-nn.log' 2> /dev/null | egrep '^\[' | grep '?token' | awk -F'=' '{print $2}'
+```
+
+Save this value as you will need it to login to your notebook.
+
+2. Create an ssh tunnel to your notebook
+
+Your ```jupyter-notebook-nn.log``` file contains the gcloud command you run to create an ssh
+tunnel to your notebook. You can run it with this command.
+
+Substitute your notebook's job number for *nn* in this command to create an ssh tunnel to your notebook.
+
+```
+$(gcloud compute ssh google1-login1 --command 'cat jupyter-notebook-nn.log' 2> /dev/null | egrep '^gcloud')
+```
+
+## Connect to your Notebook
+
+Click on the <walkthrough-web-preview-icon/> and choose the ```Preview on port 8080``` option to
+connect to your notebook. 
+
+This will open a new browser window that connects to your notebook via
+the ssh tunnel you setup in the previous step. 
+
+![Jupyter Login Screen](jupyter_login.png)
```

Comment:
Image is showing as broken in the sidebar.

Reply:
Yeah, not sure how/whether those links will show up in Tutorial mode. Will work that out with the Community repo owner when he does his review.